javascript/src_js/wrapAsm.d.ts

```diff
@@ -156,7 +156,7 @@ export type Node = {
   setWidth(width: number | string): void,
   setWidthAuto(): void,
   setWidthPercent(width: number): void,
-  unsetMeasureFun(): void,
+  unsetMeasureFunc(): void,
```

Comment:
@cc-bbohlender could you create another PR with this fix? it is not blocked

Reply:
@jeetiss done in here https://github.com/facebook/yoga/pull/1231